components/sup/src/main.rs

```diff
@@ -156,6 +220,69 @@ fn sub_config(m: &ArgMatches) -> Result<()> {
     Ok(())
 }
 
+fn sub_load(m: &ArgMatches) -> Result<()> {
+    if m.is_present("VERBOSE") {
+        sup::output::set_verbose(true);
+    }
+    if m.is_present("NO_COLOR") {
+        sup::output::set_no_color(true);
+    }
+    let cfg = mgrcfg_from_matches(m)?;
+    let ident_or_artifact = m.value_of("PKG_IDENT_OR_ARTIFACT").unwrap();
+    let ident = if Path::new(ident_or_artifact).is_file() {
```

Comment:
I don't think we need to preserve the artifact-handling here, probably only in the dev-friendly `hab-sup start` case.